go/storage/mkvs/db/badger/badger_test.go

```diff
@@ -0,0 +1,235 @@
+package badger
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-core/go/common"
+	"github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
+	"github.com/oasisprotocol/oasis-core/go/storage/api"
+	"github.com/oasisprotocol/oasis-core/go/storage/mkvs"
+	"github.com/oasisprotocol/oasis-core/go/storage/mkvs/checkpoint"
+	"github.com/oasisprotocol/oasis-core/go/storage/mkvs/writelog"
+)
+
+var (
+	nodePrefix = nodeKeyFmt.Encode()
+
+	logPrefix = multipartRestoreNodeLogKeyFmt.Encode()
+
+	testNs = common.NewTestNamespaceFromSeed([]byte("badger nodedb test ns"), 0)
+
+	dbCfg = (&api.Config{
+		Namespace:    testNs,
+		MaxCacheSize: 16 * 1024 * 1024,
+		NoFsync:      true,
+		MemoryOnly:   true,
+	}).ToNodeDB()
+
+	testValues = [][]byte{
+		[]byte("colorless green ideas sleep furiously"),
+		[]byte("excepting understandable chairs piously"),
+		[]byte("at the prickle for rainbow hoovering"),
+	}
+)
+
+type keySet map[string]struct{}
+
+type test struct {
+	require  *require.Assertions
+	ctx      context.Context
+	badgerdb *badgerNodeDB
+	restorer checkpoint.Restorer
+	ckMeta   *checkpoint.Metadata
+	ckNodes  keySet
+}
+
+func fillDB(ctx context.Context, require *require.Assertions, values [][]byte, nodedb api.NodeDB) api.Root {
```

Comment:
Just to be consistent (also elsewhere).
```suggestion
func fillDB(ctx context.Context, require *require.Assertions, values [][]byte, ndb api.NodeDB) api.Root {
```